minidump-stackwalk/stackwalker.cc

```diff
@@ -1452,6 +1450,11 @@ int main(int argc, char** argv)
   ProcessResult result =
     minidump_processor.Process(&minidump, &process_state);
 
+  int minidump_width = 64;
```

Comment:
This defaults `minidump_width` to 64. In the case of a processing error, I'm pretty sure stackwalk doesn't spit out any addresses, so it's kind of moot.